packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -645,6 +645,7 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
             let reqStToRespEndTime: number | undefined; // responseEnd - requestStart
             let networkTime: number | undefined; // responseEnd - startTime
             const spReqDuration = response.headers.get("sprequestduration");
+            const msEdge = response.headers.get("x-msedge-ref");
```

Comment:
Could you please put some comment on what's tracked in this header? Name does not tell me anything.

Reply:
I assume there is no PII involved, right?

Reply:
No PII is involved, its just tracking what machine, what data center and what time this request came in. Ex: Ref A: C0134B2202FF4A5DA1A649AF8B431473 Ref B: CO1EDGE0206 Ref C: 2020-11-12T00:56:35Z

Reply:
Ideally we should not add this info to our telemetry. It will increase cost of telemetry, while same information should be possible to retrieve from server logs, using spo correlation ID. If it's not tracked on server, can we instead implement it on spo side?

Reply:
+@ram-ramasubramanian to double click

Reply:
@sunnyhuang2008  Also please add a comment about the header as vlad suggested.

Reply:
done